rulesets/java/jpinpoint-rules.xml

```diff
@@ -324,10 +324,12 @@ class Good {
 ])]//MethodCall[@MethodName = ancestor::MethodDeclaration/@Name][not(VariableAccess or FieldAccess or SuperExpression or TypeExpression or MethodCall or AmbiguousName)]
 (: only if the number of parameters match: not an overloaded method with difference in numbers :)
 [ArgumentList/@Size = ancestor::MethodDeclaration/FormalParameters/@Size]
-(: not if a parameter is used in the call with calling a method on it :)
-[not(ArgumentList//MethodCall/VariableAccess/@Name = ancestor::MethodDeclaration/FormalParameters//VariableId/@Name)]
+(: not if a parameter is used in the call with calling a method on it or with it as param :)
```

Comment:
bit cryptic: "or with it as param" -> what does it mean?

Reply:
like getId(param) besides param.getId()